tests/settings/SettingTest.py

```diff
@@ -44,6 +50,15 @@ def test_url(self):
             uut = Setting("key", "abc")
             url(uut)
 
+    def test_glob_list(self):
+        abspath = os.path.abspath(".")
+        # Need to escape backslashes since we use list conversion
+        self.uut = Setting("key", "., " + abspath.replace("\\", "\\\\"),
+                           origin="test (1)" + os.path.sep + "somefile")
+        self.assertEqual(glob_list(self.uut),
```

Comment:
Something like:

```
self.assertEqual(
    glob_list(self.uut),
    [os.path.abspath(os.path.join("test [(]1[)]", ".")), abspath])
```
would probably be easier to read, but not important and not necessary. If youre modifying the `separator` issue, then do this